websocket-tcp-proxy/src/main.rs

```diff
@@ -16,19 +17,24 @@ use tower_http::trace::{DefaultMakeSpan, TraceLayer};
 
 #[tokio::main]
 async fn main() {
+    println!("Running websocket-tcp-proxy");
     tracing_subscriber::fmt::init();
 
     let app = Router::new().route("/v1/:ip/:port", get(ws_handler)).layer(
         TraceLayer::new_for_http().make_span_with(DefaultMakeSpan::default().include_headers(true)),
     );
 
-    // TODO let this be configurable
-    let addr = SocketAddr::from(([127, 0, 0, 1], 3001));
+    let port = match env::var("PORT") {
```

Comment:
should make this something like `MUTINY_PROXY_PORT` so it doesn't accidentally overlap with something else

Reply:
good idea